sum/contracts/assembly/contracts/run.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  Address,
+  Storage,
+  call,
+  callerHasWriteAccess,
+} from '@massalabs/massa-as-sdk';
+import { NoArg, Args } from '@massalabs/as-types';
```

Comment:
This should be seen and bloc by the linter in the CI. Can you create an issue to check why the Ci is not working ?

Reply:
Done